README.md

```diff
@@ -65,36 +64,38 @@ Note that when you set a custom path for a component, the global path prefix is
 
 Any custom configuration should be set before including any other components in your product's sass/js bundles.
 
-1. Global path prefix: `$o-assets-global-path` variable (in Sass) and `setGlobalPathPrefix` method (in JavaScript).  Default is '/bower_components/'.
+1. Global path prefix: `$o-assets-global-path` variable (in Sass) and `setGlobalPathPrefix` method (in JavaScript). Default is '/bower_components/'.
 1. component path: This defaults to the name of the component and is set using methods that accept a map of component names and paths:
 
 #### Sass
 
 ```scss
-@include oAssetsSetcomponentPaths((
-	'o-example': '/assets/example'
-));
+@include oAssetsSetModulePaths(
+	(
+		'o-example': '/assets/example'
+	)
+);
```

Comment:
Is the indentation change here intentional?

Reply:
hmm! i don't think so ! i'll fix that